homeassistant/components/android_ip_webcam/config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Config flow for Android IP Webcam integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from pydroid_ipcam import PyDroidIPCam
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SCAN_INTERVAL,
+    CONF_TIMEOUT,
+    CONF_USERNAME,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DEFAULT_PORT, DEFAULT_TIMEOUT, DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
```

Comment:
Do we need a timeout option or can we just use the default?

Reply:
The default should be fine. I added it here because it is mentioned in the yaml config. Should we also remove it when importing?

Reply:
Let's remove it completely and see if we get any issues reported. We can tweak the default if there are any reports.